econml/inference/_inference.py

```diff
@@ -462,9 +471,12 @@ def fname_transformer(x):
         else:
             def fname_transformer(x):
                 return x
+        # d_t=1 here since we measure the effect across all Ts
+        # Omit treatment name argument to fall back to the default
         return NormalInferenceResults(d_t=1, d_y=self.d_y, pred=coef, pred_stderr=coef_stderr,
                                       inf_type='coefficient', fname_transformer=fname_transformer,
-                                      **self._input_names)
+                                      feature_names=self._input_names["feature_names"],
```

Comment:
I don't like that this is accessing the _input_names dict directly. This hsould instead be accessing the public cate_feature_names and cate_output_names.

Reply:
Fixed!